src/index.html

```diff
@@ -1,12 +1,322 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a href="#" class="header__link">
+        <img src="images/logo.png" alt="logo-img">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li>
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+
+          <li>
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li>
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+
+          <li>
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+
+          <li>
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+
+          <li>
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="catalog">
+        <article class="card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="Apple A1419 iMac 27"
+            class="card__img"
+          >
+          <div class="card__info">
+            <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+            <p class="card__code">Product code: 195434</p>
+            <div class="card__stars">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__reviews">Reviews: 5</p>
+            </div>
+            <div class="card__price">
+              <p class="card__value">Price:</p>
+              <p class="card__sum">$2,199</p>
+            </div>
+            <a
+            href="#"
+            class="button"
+            data-qa="card-hover"
+          >
+            buy
+            </a>
```

Comment:
Pay attention on code style
```suggestion
            <a
              href="#"
              class="button"
              data-qa="card-hover"
            >
              buy
            </a>
```